pkg/migrations/mls/20240109001927_add-messages.up.sql

```diff
@@ -0,0 +1,27 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE TABLE group_messages (
+    id BIGSERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    group_id VARCHAR(200) NOT NULL,
```

Comment:
Think this is fine to store as text, but at the `libxmtp` and `openmls` level `group_id`s are always byte arrays. We can stringify in the node, obviously, but it might be simpler to just keep them as byte arrays all the way.

Reply:
Installation ID too or just group ID?

Reply:
They're both byte arrays from the client's perspective. The installation id is actually a public key.